src/workflow/units/execution/espresso.py

```diff
@@ -13,6 +18,22 @@ class EspressoExecutionUnit(BaseExecutionUnit):
     def __init__(self, config, work_dir):
         super(EspressoExecutionUnit, self).__init__(config, work_dir)
 
+    @property
+    def stdout_file(self):
+        stdout_file = super(EspressoExecutionUnit, self).stdout_file
+        if os.path.exists(stdout_file): return stdout_file
+        return find_file_with_pattern(self.work_dir, ESPRESSO_EXECUTABLE_NAME_REGEX)
+
+    @property
+    def version(self):
+        """
+        Returns the application version used in the unit.
+
+        Returns:
+             str
+        """
+        return "5.4.0"
```

Comment:
I don't understand the logic - why do we assume it's always that version? We can either set it to None/NotImplemented/Unknown or extract the version from output ...

Reply:
Fixed.